src/main/java/com/google/cloud/tools/appengine/cloudsdk/AppEngineComponentsNotInstalledException.java

```diff
@@ -0,0 +1,11 @@
+package com.google.cloud.tools.appengine.cloudsdk;
+
+import com.google.cloud.tools.appengine.api.AppEngineException;
+
+public class AppEngineComponentsNotInstalledException extends AppEngineException {
```

Comment:
also I haven't checked the styleguide recently, is the javadoc no longer required for all public classes?=